client/src/components/EditProfile/ProfileForm/ProfileForm.tsx

```diff
@@ -0,0 +1,181 @@
+import { useState, ChangeEvent } from 'react';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Box from '@material-ui/core/Box';
+import { Formik, FormikHelpers } from 'formik';
+import * as Yup from 'yup';
```

Comment:
You imported this package, which should be used for form validation. If not meant to be used here, it should be removed.